internal/flags/flags.go

```diff
@@ -77,7 +79,10 @@ func init() {
 		log.Print(usage)
 		pflag.PrintDefaults()
 	}
-
+	offline, err := strconv.ParseBool(env.GetEnvOrDefault("TASK_OFFLINE", "false"))
+	if err != nil {
+		offline = false
+	}
```

Comment:
Since 1.22 we can use the standard `cmp` package for this.

```suggestion
	offline, err := strconv.ParseBool(cmp.Or("TASK_OFFLINE", "false"))
	if err != nil {
		offline = false
	}
```

Reply:
Oh I didn't know! Thanks